src/tag/tag.controller.ts

```diff
@@ -1,12 +1,12 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, Query } from '@nestjs/common';
 import { TagService } from './tag.service';
 
 @Controller('tag')
 export class TagController {
   constructor(private readonly tagService: TagService) {}
 
   @Get()
-  async getAllTags() {
-    return await this.tagService.getAllTags();
+  async searchTags(@Query('search') search: string) {
```

Comment:
Add also take Query (default should be 10) which will indicate result limit

Reply:
Done